src/main/java/dev/jbang/cli/KeyValueConsumer.java

```diff
@@ -20,17 +19,15 @@ public void consumeParameters(Stack<String> args, CommandLine.Model.ArgSpec argS
 		Matcher m = p.matcher(arg);
 		if (m.matches()) {
 
-			Map<String, Optional<String>> kv = argSpec.getValue();
+			Map<String, String> kv = argSpec.getValue();
```

Comment:
how come not optional anymore ? how do you know if it was specified but no value ?

Reply:
Just like before but using `null` instead of `Optional.empty()`. The example you giove would have been `Optional.empty()` and `Optional("")`, now it is `null` and `""`. I just needed this to be easily (de)serializable to/from JSON. It's easier to make this change than to add more custom (de)serializers. Also, all the code became easier because we were doing `value.orElse(null)` all over the place.